src/core/modules/Planet.cpp

```diff
@@ -1768,7 +1768,7 @@ QVector<const Planet*> Planet::getCandidatesForShadow() const
 	return res;
 }
 
-void Planet::computePosition(const StelObserver *observer, const double dateJDE, const Vec3d &aberrationPush)
+void Planet::computePosition(const StelObserver *observer, const double dateJDE, const Vec3d &aberrationPush)  // heliocentric
```

Comment:
This comment should be part of the docs in the header.

Reply:
fixed